Jenkinsfile_k8s

```diff
@@ -1 +1,5 @@
-parallelDockerUpdatecli([imageName: 'account-app', rebuildImageOnPeriodicJob: false, buildDockerConfig: [targetplatforms: 'linux/amd64,linux/arm64']])
+buildDockerAndPublishImage('account-app', [
+    rebuildImageOnPeriodicJob: false,
+    automaticSemanticVersioning: true,
+    targetplatforms: 'linux/amd64,linux/arm64'
+])
```

Comment:
We should merge these 2 pipelines into a single one to ensure the same set of parameters is applied to both CI and CD process (and avoid CD issues which could have been caught by CI).

=> WDYT about the method you used (can't remember where though): `disablePublication: !infra.isInfra()` ?

Reply:
This is the next step.
First I needed to be sure that my pipeline change worked.